randovania/games/prime2/json_data/Agon Wastes.txt

```diff
@@ -2490,7 +2500,9 @@ Extra - in_dark_aether: False
                   Morph Ball Bomb
                   Any of the following:
                       Bomb Jump (Hypermode)
-                      Space Jump Boots and Bomb Space Jump (Intermediate) and Standable Terrain (Intermediate)
+                      All of the following:
+                          # https://www.youtube.com/watch?v=PHjXnyJA2is
+                          Space Jump Boots and Bomb Space Jump (Intermediate) and Standable Terrain (Intermediate)
```

Comment:
Was this video meant to be linked in this group or added as a separate NSJ trick?

Reply:
To be linked in this group. I thought about asking if it was alright to remove the Hypermode Bomb Jump, but decided against it. There might be someone that wants to do this trick, but does not want to set Bomb Jump to Hypermode.